Makefile

```diff
@@ -18,10 +18,12 @@ ifeq ($(BUILD_TYPE),release)
 	PG_LDFLAGS = $(LDFLAGS)
 	# Unfortunately, `--profile=...` is a nightly feature
 	CARGO_BUILD_FLAGS += --release
+	NEON_CARGO_ARTIFACT_TARGET_DIR = $(ROOT_PROJECT_DIR)/target/release
```

Comment:
Bummer that there's no reliable way to get this from `cargo`. We could parse the `cargo metadata` output, but hardcoding it seems find until `cargo` provides a canonical way to detect this.

Reply:
https://doc.rust-lang.org/cargo/reference/config.html#buildtarget

You can specify it, though. And if you want to make it consistent with cargo, specify the same exactly thing in both places :-)

Reply:
I don't know. It would be nice if this worked with non-default toolchain. I've been using nightly toolchain myself, and that doesn't affect the target path, i.e. it's still `target/release` or `target/debug`. I don't know if e.g. cross-compilation would use a different path.

Anyway, I think we can leave this like this for now. It's likely that there will be an easier way to get the path from `cargo` in a future version, and we can switch to that then.

Reply:
I think https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#artifact-dir would be just what we want.